osprey/utils.py

```diff
@@ -161,7 +161,19 @@ def convolve_config_handler(
         )
         processed["INPUT_FORCINGS"]["DATL_FILE"] = input_forcings.split("/")[-1]
 
-        return processed
+        return rvic_config_validator(processed)
 
     except KeyError:
         raise ProcessError("Invalid config key provided")
+
+
+def rvic_config_validator(cfg):
+    return {
+        section: {
+            key: cfg[section][key]
+            if type(cfg[section][key]) != list or len(cfg[section][key]) > 1
+            else cfg[section][key][0]
+            for key in cfg[section].keys()
+        }
+        for section in cfg.keys()
+    }
```

Comment:
@nikola-rados Surprisingly, storing `config_template`s in a separate file caused a problem, and it is also very odd. Only dictionary values in the `HISTORY` section in `convolve_config_template` are changed when the dictionary is imported in `wps_convolution` and `wps_full_rvic`. This function `rvic_config_validator` is to re-convert the values to they original data types.